src/packages/core/content-type/workspace/views/design/content-type-design-editor-property.element.ts

```diff
@@ -81,8 +83,32 @@ export class UmbContentTypeDesignEditorPropertyElement extends UmbLitElement {
 	@state()
 	private _dataTypeName?: string;
 
-	@state()
-	private _aliasLocked = true;
+	#autoGenerateAlias = true;
+
+	#checkAliasAutoGenerate(unique: string | undefined) {
+		if (unique === this.#propertyUnique) return;
+		this.#propertyUnique = unique;
+		this.#autoGenerateAlias;
+
+		if (this.#context.getAlias()) {
+			this.#autoGenerateAlias = false;
+		}
+
+		/* TODO: Do we know if we are loading an existing property (do not auto generate alias) or added a new property (auto-generate only if matching expected alias)? 
```

Comment:
I would say please check how the existing Backoffice works, make sure to make the experience match. :-)